platform-sdk/base-apps/swirlds-base-sample/src/main/resources/log4j2.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration packages="pl.tkowalcz.tjahzi.log4j2">
+<Loggers>
+  <Root level="TRACE">
+    <AppenderRef ref="Console"/>
+    <AppenderRef ref="Loki"/>
+  </Root>
+</Loggers>
+
+<appenders>
+  <Console name="Console" target="SYSTEM_OUT">
+    <PatternLayout pattern="%d{yyyy-MM-dd HH:mm:ss.SSS} %-5p %-4L %c{1} - %m{nolookups}%n"/>
+  </Console>
+
+  <Loki name="Loki" bufferSizeMegabytes="64">
+    <host>localhost</host>
+    <port>3100</port>
+
+    <ThresholdFilter level="ALL"/>
+    <PatternLayout>
+      <pattern>%d{yyyy-MM-dd HH:mm:ss.SSS} %-8sn %-5p %-16marker &lt;%t&gt; %c{1}: %msg{nolookups}%n</pattern>
+    </PatternLayout>
```

Comment:
```suggestion
```

I guess this is not needed

Reply:
According to the documentation: https://github.com/tkowalcz/tjahzi/blob/master/log4j2-appender/README.md, it seems that it is, haven't tried without it but I would keep it only for making sure we have all what is expected